test/CheckoutSdkTest/Payments/Hosted/PaymentsClientTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Checkout.Common;
+using Moq;
+using Shouldly;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Checkout.Payments.Hosted
+{
+    public class PaymentsClientTest : AbstractPaymentsIntegrationTest
+    {
+        private const string HostedPayments = "/hosted-payments";
+        private const string Reference = "ORD-1234";
+        private const string ValidDefaultSk = "sk_test_fde517a8-3f01-41ef-b4bd-4282384b0a64";
```

Comment:
UnitTestFixture already contains `ValidDefaultSk` as a constant.

Reply:
I had to inherit from `AbstractPaymentsIntegrationTest ` because I need the `CreateHostedPaymentRequest ` method (helps me save 55 lines of code).